.travis.yml

```diff
@@ -0,0 +1,24 @@
+# You must sign into travis-ci.org and set the commit hook on your project for travis to
+# run on your project. The secure: variable must be generated by running `travis encrypt`
+# on a github oauth key that you can generate using curl.
+
+sudo: false
+language: node_js
+node_js:
+  - 'stable'
+addons:
+  firefox: latest
```

Comment:
It [looks like](https://docs.travis-ci.com/user/firefox/#Version-aliases) there's a `latest-beta` alias that corresponds to the Firefox Beta release. Does `.travis.yml` let you supply a list of aliases here, or are you forced to choose only one of `latest` and `latest-beta`?